packages/vm/src/cache.rs

```diff
@@ -203,7 +206,11 @@ where
             self.stats.hits_fs_cache += 1;
             let instance =
                 Instance::from_module(&module, backend, options.gas_limit, options.print_debug)?;
-            self.memory_cache.store(checksum, module)?;
+            // Serializes module to get the size
+            // FIXME? Use just an estimate (average) of the module size
+            // let size = ESTIMATED_MODULE_SIZE.0;
+            let size = module.serialize()?.len();
```

Comment:
We could return this value as part of the `self.fs_cache.load(` call, which can easily know the file size.

Reply:
Good idea, I'll do this.